production/scrapers/georgia_youth.R

```diff
@@ -0,0 +1,91 @@
+source("./R/generic_scraper.R")
+source("./R/utilities.R")
+
+georgia_youth_pull <- function(x){
+    xml2::read_html(x)
+}
+
+georgia_youth_restruct <- function(x){
+    x %>%
+        rvest::html_nodes('table') %>%
+        .[[1]] %>%
+        rvest::html_table()
+}
+
+georgia_youth_extract <- function(x){
+    check_names(x,
+                c("DJJ Location",
+                  "Staff\nConfirmed",
+                  "Staff\nRecovered",
+                  "Youth\nConfirmed",
+                  "Youth\nRecovered"))
+    
+    clean <- x %>%
+        select(Name = "DJJ Location", 
+               Residents.Confirmed = `Youth\nConfirmed`,
+               Residents.Recovered = `Youth\nRecovered`,
+               Staff.Confirmed = `Staff\nConfirmed`,
+               Staff.Recovered = `Staff\nRecovered`
+               ) %>%
+        mutate(Name = str_c(toupper(Name), " YOUTH")) %>%
+        filter(Name != "TOTAL YOUTH")
```

Comment:
Feel free to ignore, but it might be safer to use something like this here, incase they change the name to just `Total` or `Total Juvenile` or something that isn't exactly `Total Youth`. 
```
filter(!str_detect(Name, "(i?)TOTAL")) 
```